tool/jt.rb

```diff
@@ -107,15 +106,20 @@ def self.find_graal_javacmd_and_options
     [javacmd, vm_args.map { |arg| "-J#{arg}" } + options]
   end
 
-  def self.find_mx(graal_home)
-    sibling_mx = File.expand_path("../mx/mx", graal_home)
-    sibling_parent_mx = File.expand_path("../../mx/mx", graal_home)
-    if File.executable?(sibling_mx)
-      sibling_mx
-    elsif File.executable?(sibling_parent_mx)
-      sibling_parent_mx
+  def self.which(binary)
+    ENV["PATH"].split(File::PATH_SEPARATOR).each do |dir|
+      path = "#{dir}/#{binary}"
+      return path if File.executable? path
+    end
+    nil
+  end
+
+  def self.find_mx
```

Comment:
I was thinking more along the lines of what we already do with `mx sforcemimports`. I'm just a bit worried that if we transparently fetch mx, no one is going to know how to update it. But, I agree having to hit git origin every time would be annoying.